### ISSUE TITLE
polygon hasher has latitude and longitude  swapped

### ISSUE THREAD

**Issue by Karlpietsch** [Postlagerkarte/geohash-dotnet#47]
```csharp
private Geometry GeohashToPolygon(string geohash)
{
    var corners = geohasher.GetBoundingBox(geohash);

    var coordinates = new Coordinate[]
    {
        new Coordinate(corners[0], corners[2]),
        new Coordinate(corners[1], corners[2]),
        new Coordinate(corners[1], corners[3]),
        new Coordinate(corners[0], corners[3]),
        new Coordinate(corners[0], corners[2]),
    };

    var geometryFactory = new GeometryFactory();

    return geometryFactory.CreatePolygon(coordinates);

}
```

is incorrect the latitude and longitude are in the wrong order

```csharp
public Coordinate(double x, double y);
```
```csharp
// <returns>bounding box as double[] containing latInterval[0], latInterval[1], lonInterval[0], lonInterval[1]</returns>
public double[] GetBoundingBox(string geohash)
```

that is  corners[0] and corners[1] are latittude/Y coordinates and  corners[2] and corners[3] are longitude/X coordinates

code needs to be 

```csharp
private Geometry GeohashToPolygon(string geohash)
{
    var corners = geohasher.GetBoundingBox(geohash);

    var coordinates = new Coordinate[]
    {
        new Coordinate(corners[2], corners[0]),
        new Coordinate(corners[2], corners[1]),
        new Coordinate(corners[3], corners[1]),
        new Coordinate(corners[3], corners[0]),
        new Coordinate(corners[2], corners[0]),
    };

    var geometryFactory = new GeometryFactory();

    return geometryFactory.CreatePolygon(coordinates);

}
```

or 
```csharp
private Geometry GeohashToPolygon(string geohash)
{
    var corners = geohasher.GetBoundingBox(geohash);

    var coordinates = new Coordinate[]
    {
        new Coordinate {Y = corners[0], X = corners[2] },
        new Coordinate {Y = corners[1], X = corners[2] },
        new Coordinate {Y = corners[1], X = corners[3] },
        new Coordinate {Y = corners[0], X = corners[3] },
        new Coordinate {Y = corners[0], X = corners[2] },
    };


    var geometryFactory = new GeometryFactory();

    return geometryFactory.CreatePolygon(coordinates);

}
```

**Comment by cs-util** [comment]
@Karlpietsch could you make a fork of https://github.com/cs-util/geohash-dotnet and add a unit test that ensures this fix wont break in the future? I made the tests work again in there

**Comment by Postlagerkarte** [comment]
fixed in v2.0
